.mocharc.json

```diff
@@ -11,5 +11,6 @@
   "recursive": true,
   "failZero": true,
   "reporter": "test/tools/mongodb_reporter.js",
-  "color": true
-}
+  "color": true,
+  "timeout": 0
```

Comment:
Unsquashing test failures in the Dockerfile (see my other comment) reveals test timeouts on arm builds.  These docker containers are just really slow.

Reply:
Now GH will timeout the run eventually right? this is fine, but just curious how long one would go before the catch a hanging bug or something, maybe a high value is better than forever, do you know what GH's timeout is?

Reply:
6 hours, I think